types/react-with-styles/lib/WithStylesContext.d.ts

```diff
@@ -1,4 +1,4 @@
 import { Context } from "react";
 
-declare const context: Context<{ stylesInterface: any, stylesTheme: any }>;
-export default context;
\ No newline at end of file
+declare const context: Context<{ stylesInterface: any, stylesTheme: any, direction?: any }>;
```

Comment:
This seems unrelated

Reply:
The context provider default value can get a `direction` property too. This is to allow that.

Reply:
That seems unrelated to "Make the default theme empty so users can extend it"